test/integration/modules/vso-helm/main.tf

```diff
@@ -22,9 +22,19 @@ resource "helm_release" "vault-secrets-operator" {
     name  = "defaultAuthMethod.enabled"
     value = var.enable_default_auth_method
   }
-  set {
-    name  = "defaultAuthMethod.namespace"
-    value = var.operator_namespace
+  dynamic "set" {
+    for_each = var.k8s_auth_default_mount != "" ? [""] : []
+    content {
+      name  = "defaultAuthMethod.mount"
+      value = var.k8s_auth_default_mount
+    }
+  }
```

Comment:
Just making sure I understand, this sets `k8s_auth_default_mount` in the chart values if it's not blank?

Reply:
yes

Reply:
Updated with ef59401fddfa1f0cf01bb22508792a8e3840d6b8